scripts/install/ceres_install.bash

```diff
@@ -2,6 +2,7 @@
 set -e  # exit on first error
 UBUNTU_VERSION=`lsb_release --release | cut -f2`
 SRC_PREFIX_PATH="/usr/local/src/"
```

Comment:
No. I am just curious.  Good to know the standard!  As all the 3rd-party libraries in `/usr/local/src` have been installed and the `build` folders are not needed, is it a good practice to delete the `build` folder to save space?   The build folders can be really big after several libraries installed.